packages/client/src/navigation/index.ts

```diff
@@ -408,6 +411,15 @@ export function goToFieldAgentHomeTab(tabId: string) {
   }
 }
 
+export function goToPageNavigation(event: string, section: string) {
```

Comment:
The name for this function seems kinda irrelevant. Maybe `goToFormConfigWizard` would have been better?